tests/test_storage.py

```
@@ -946,3 +889,147 @@ async def test_get_known_branches(db_path: Path) -> None:
+        )
+        assert project is not None
+        assert project["total_cost_usd"] == 1.23
+    finally:
+        storage.DB_PATH = orig_db
+        storage.PROJECTS_DIR = orig_projects
+        storage.DATA_DIR = orig_data
+
+
+async def test_get_total_cost(tmp_path: Path) -> None:
+    """get_total_cost sums total_cost_usd across all variants."""
+    import docsfy.storage as storage
```

Ruff (0.15.12)

[warning] Use `from docsfy import storage` in lieu of alias

Replace with `from docsfy import storage`

(PLR0402)